packages/hint-no-vulnerable-javascript-libraries/src/types.ts

```diff
@@ -6,26 +6,13 @@ export type Library = {
     npmPkgName: string;
 };
 
-/** A snyk.io vulnerability report. */
+/** A snyk.io vulnerability report (just the pieces used by webhint). */
 
 export type Vulnerability = {
-    title: string;
-    moduleName: string;
```

Comment:
Can you give an example of `moduleName`?  Maybe we need it for when we analyze `package.json`?

Reply:
This appears to be the same as `packageName`. I haven't found an example where it differs yet.

Reply:
@molant Also, we can always bring any particular property back if we decide we want to use it. This is about optimizing for what we *currently* use. I added a comment to the jsdoc for `Vulnerability` to make it clear what needs to change if we decide to include anything else in the future.

Any other feedback?

Reply:
Not from my side :)